project/store/src/main/java/org/fjala/prog102/store/repositories/ProductRepository.java

```diff
@@ -0,0 +1,8 @@
+package org.fjala.prog102.store.repositories;
+
+import org.fjala.prog102.store.models.Product;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface ProductRepository extends JpaRepository<Product, Long> {
```

Comment:
I think we should use CrudRepository

Reply:
So I think we just need Crud repository, I'll change it on the next PR